courses/free-courses-en.md

```diff
@@ -339,6 +339,7 @@
 
 * [ASP.NET Core 2.2 & 3 REST API Tutorial](https://youtube.com/playlist?list=PLUOequmGnXxOgmSDWU7Tl6iQTsOtyjtwU&si=3XHU6BnqwxePAt54) - Nick Chapsas
 * [Building a microservice architecture with ASP.NET Core - Gill Cleeren - NDC London 2022](https://www.youtube.com/watch?v=SR53SKIUYPA) - Gill Cleeren
+* [Build mobile and desktop apps with .NET MAUI](https://learn.microsoft.com/en-us/training/paths/build-apps-with-dotnet-maui) - James Montemagno (Microsoft)
```

Comment:
Swap line 342 and 341. These are sorted in alphabetical order. Swapping them will resolve the issue. For further information see [here](https://github.com/vhf/remark-lint-alphabetize-lists#remark-lint-alphabetize-lists-)

Reply:
@rng70 thanks. Let me do that, I was wondering what the problem was and couldn't see it.